code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -948,3 +948,30 @@
 				to_chat(M, "<span class='warning'>Your missing arm aches from wherever you left it.</span>")
 				M.emote("sigh")
 	return ..()
+
+	/datum/reagent/iodomethane
```

Comment:
Wrong indention.

Select all these new lines in your local editor and press SHIFT plus TAB to remove the unneeded extra tab space.

Reply:
Also turn these reagents into subtypes of `/datum/reagent/toxin` and remove the `var/` prefix from their toxpwr.